packages/react-native-renderer/src/ReactFabricRenderer.js

```diff
@@ -0,0 +1,321 @@
+
+type Node = Object;
+type ChildSet = Object;
+type Container = number;
+type Instance = {
+  node: Node,
+  canonical: ReactFabricHostComponent,
+};
+type Props = Object;
+type TextInstance = {
+  node: Node,
+};
+
+const ReactFabricRenderer = ReactFiberReconciler({
+  appendInitialChild(
+    parentInstance: Instance,
+    child: Instance | TextInstance,
+  ): void {
+    FabricUIManager.appendChild(parentInstance.node, child.node);
+  },
+
+  createInstance(
+    type: string,
+    props: Props,
+    rootContainerInstance: Container,
+    hostContext: {},
+    internalInstanceHandle: Object,
+  ): Instance {
+    const tag = ReactNativeTagHandles.allocateTag();
+    const viewConfig = ReactNativeViewConfigRegistry.get(type);
+
+    if (__DEV__) {
+      for (const key in viewConfig.validAttributes) {
+        if (props.hasOwnProperty(key)) {
+          deepFreezeAndThrowOnMutationInDev(props[key]);
+        }
+      }
+    }
+
+    const updatePayload = ReactNativeAttributePayload.create(
+      props,
+      viewConfig.validAttributes,
+    );
+
+    const node = FabricUIManager.createNode(
+      tag, // reactTag
+      viewConfig.uiViewClassName, // viewName
+      rootContainerInstance, // rootTag
+      updatePayload, // props
+      internalInstanceHandle, // internalInstanceHandle
+    );
+
+    const component = new ReactFabricHostComponent(tag, viewConfig, props);
+
+    return {
+      node: node,
+      canonical: component,
+    };
+  },
+
+  createTextInstance(
+    text: string,
+    rootContainerInstance: Container,
+    hostContext: {},
+    internalInstanceHandle: Object,
+  ): TextInstance {
+    const tag = ReactNativeTagHandles.allocateTag();
+
+    const node = FabricUIManager.createNode(
+      tag, // reactTag
+      'RCTRawText', // viewName
+      rootContainerInstance, // rootTag
+      {text: text}, // props
+      internalInstanceHandle, // instance handle
+    );
+
+    return {
+      node: node,
+    };
+  },
+
+  finalizeInitialChildren(
+    parentInstance: Instance,
+    type: string,
+    props: Props,
+    rootContainerInstance: Container,
+  ): boolean {
+    return false;
+  },
+
+  getRootHostContext(): {} {
+    return emptyObject;
+  },
+
+  getChildHostContext(): {} {
+    return emptyObject;
+  },
+
+  getPublicInstance(instance) {
+    return instance.canonical;
+  },
+
+  now: ReactNativeFrameScheduling.now,
+
+  prepareForCommit(): void {
+    // Noop
+  },
+
+  prepareUpdate(
+    instance: Instance,
+    type: string,
+    oldProps: Props,
+    newProps: Props,
+    rootContainerInstance: Container,
+    hostContext: {},
+  ): null | Object {
+    const viewConfig = instance.canonical.viewConfig;
+    const updatePayload = ReactNativeAttributePayload.diff(
+      oldProps,
+      newProps,
+      viewConfig.validAttributes,
+    );
+    // TODO: If the event handlers have changed, we need to update the current props
+    // in the commit phase but there is no host config hook to do it yet.
```

Comment:
Do you plan to create a spill-over issue or do it here? Easy to forget something like this.